### PR TITLE
Link Clean Up

### PR DESCRIPTION
**Pull request by KieranRobson**
Cleaned up and updated URLs
There are plenty more but as this is my first PR - left it at these 5!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -237,7 +237,6 @@ Table of Contents
   * [Glitterly](https://glitterly.app/) - Programatically generate dynamic images from base templates. Restful API and nocode integrations. Free tier comes with 50 images/month and 5 templates.
   * [GoodData](https://www.gooddata.com/) - Data as a Service - Create interactive and insightful dashboards. Free tier comes with 5 workspaces and 100 MB/workspace.
   * [Hex](https://hex.tech/) - a collaborative data platform for notebooks, data apps and knowledge libraries. Free community version with up to 3 authors and 5 projects. One compute profile per author with 4GB RAM.
-  * [Hookbin](https://hookbin.com/) - Create unique (public or private) endpoints to collect, parse, and inspect HTTP requests. Inspect headers, body, query strings, cookies, uploaded files, etc. Useful for testing/inspecting webhook. Similar to RequestBin, and Webhook.site.
```
Comment (ripienaar) [inline README.md:240]: this is still up and working

File: README.md [hunk 2]
```diff
@@ -297,7 +296,7 @@ Table of Contents
   * [ScraperBox](https://scraperbox.com/) — Undetectable web scraping API using real Chrome browsers and proxy rotation. Use a simple API call to scrape any web page. Free plan has 1000 requests per month.
   * [ScrapingDog](https://scrapingdog.com/) — Scrapingdog handles millions of proxies, browsers and CAPTCHAs to provide you with HTML of any web page in a single API call. It also provides Web Scraper for Chrome & Firefox and a software for instant scraping demand. Free plans available.
   * [scrapinghub.com](https://scrapinghub.com) — Data scraping with visual interface and plugins. Free plan includes unlimited scraping on a shared server.
-  * [ScrapingNinja](https://www.scrapingninja.co/) — Handle JS rendering, Chrome Headless, Proxy rotation and CAPTCHAs solving all in one place. The first 1000 are free of charge, no credit card required.
+  * [ScrapingBee](https://www.scrapingbee.com//) — Handle JS rendering, Chrome Headless, Proxy rotation and CAPTCHAs solving all in one place. The first 1000 are free of charge, no credit card required.
```
Comment (ripienaar) [inline README.md:299]: Its not clear now if this is 1000 free or 1000 free a month. So I am inclined to just remove it.

**Review by ripienaar — CHANGES_REQUESTED** [review]
small changes, thanks for this!

**Comment by ripienaar** [comment]
thank you

**Comment by imba-tjd** [comment]
I singed up ScrapingBee. It offers a one-time 1000 credits expired in 14 days. A simplest request consumes 5 points. Rendering JS consumes more.

**Comment by ripienaar** [comment]
> I singed up ScrapingBee. It offers a one-time 1000 credits expired in 14 days. A simplest request consumes 5 points. Rendering JS consumes more.

Please send a PR to remove it

**Comment by imba-tjd** [comment]
No need 😂 It's actually removed in this PR. I just share some info.

**Pull request merged**